src/passes/Unsubtyping.cpp

```diff
@@ -446,6 +446,10 @@ struct Unsubtyping
   void visitTableFill(TableFill* curr) {
     noteSubtype(curr->value->type, getModule()->getTable(curr->table)->type);
   }
+  void visitTableCopy(TableCopy* curr) {
```

Comment:
It would be nice to add a test case for this in test/lit/passes/unsubtyping.wast.